src/Commands.pm

```diff
@@ -7033,6 +7033,29 @@ sub cmdAchieve {
 			$messageSender->sendAchievementGetReward($args[1]);
 		}
 
+	} elsif ($args[0] =~ /^\d+$/) {
+		if(defined($achievements{$args[0]})) {
+			# status
+			message center(" " . T("Achievement Info") . " ", 43, '-') . "\n";
```

Comment:
this is a bad design:
```
message "***";
message "***";
message "***";
```
see [quest info](https://github.com/OpenKore/openkore/blob/eb1af8dbf59655dec5517e51331c8de46da14d20/src/Commands.pm#L6806) :
```
my $msg = "***";
$msg .= "***";
message $msg;
```